src/@ionic-native/plugins/local-notifications/index.ts

```diff
@@ -125,7 +125,7 @@ export interface ILocalNotificationAction {
   /**
    * Specifies whether the action causes the app to launch in the foreground
    */
-  launch?: boolean;
```

Comment:
We should keep `launch` to maintain backwards compatibility. No need to introduce a breaking change yet.

Reply:
I agree. I'll go ahead and update that.